applications/RomApplication/python_scripts/hrom_training_utility.py

```diff
@@ -43,9 +44,15 @@ def __init__(self, solver, custom_settings):
         self.projection_strategy = settings["projection_strategy"].GetString()
         self.hrom_output_format = settings["hrom_format"].GetString()
 
+        # Rom settings files
+        self.rom_basis_output_name = custom_settings["rom_basis_output_name"].GetString()
+        self.rom_basis_output_folder = custom_settings["rom_basis_output_folder"].GetString()
+        self.rom_basis_output_folder = Path(self.rom_basis_output_folder)
+        self.rom_basis_output_name = Path(self.rom_basis_output_name)   
+
     def AppendCurrentStepResiduals(self):
         # Get the computing model part from the solver implementing the problem physics
-        computing_model_part = self.solver.GetComputingModelPart()
+        computing_model_part = self.solver.GetComputingModelPart().GetRootModelPart()
```

Comment:
```suggestion
        root_model_part = self.solver.GetComputingModelPart().GetRootModelPart()
```
To make clear which model part we're using below.

BTW, is there any specific reason for this?